# Drake physics parameters
# time_step: used in physics integration 
# viz_time_step: physics publish data time step
physics:
  frequency: 600.0 
  time_step: 0.0001
  viz_time_step: 0.001
  static_friction: 1.0
  dynamic_friction: 1.0
  penetration_allowance: 0.001
  # stiction_tolerance: 0.001
  real_time_rate: 1.0
